src/main.cpp

```diff
@@ -78,6 +82,12 @@ void InitGyro()
     #endif
 }
 
+
+/*
+* initialize the accelerometer and set the sample rate frequency 
+* Also get the base offset of the accelerometer at startup to give better
+* measurements in the GetAnglemeasurements function
+*/
```

Comment:
Again, align the '*' characters please. Capitalize! 

If you end a sentence, put a '.' character.